### PR TITLE
feat: apigw to sqs pattern (typescript)

### PR DESCRIPTION
**Pull request by ryanseee**
*Issue #, if available:*
#1134 

*Description of changes:*
This is the typescript addition to an existing serverless pattern (API Gateway - SQS). It includes an L3 construct which users could also extract and make use as they deem fit.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: apigw-sqs-cdk-typescript/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+## Testing
+
+Run the following command to send a POST request to the REST API endpoint that has been created. Note, replace {MyRESTApi} placeholder in the command with the endpoint that has been deployed. The endpoint can be found in the CloudFormation stack output. 
+
+```bash
+curl --location --request POST '{MyRESTApi}/' \
+> --header 'Content-Type: application/json' \
+> --data-raw '{ "message": "this is a test message" }'
+```
```
[the remark this answers is not in the document]
Comment (mavi888) [reply apigw-sqs-cdk-typescript/README.md:56]: I cannot see the message "this is a test message" in the received message in the queue. My message has an empty body.

**Comment by mavi888** [comment]
Hello, thanks for your PR, before I merge it please address the comment above.

Cheers,
Marcia

**Comment by ryanseee** [comment]
> Hello, thanks for your PR, before I merge it please address the comment above.
> 
> Cheers, Marcia

Hi @mavi888, I have edited the CLI command to send message to SQS and also included the command to receive message from SQS.

**Comment by mavi888** [comment]
Thanks for submitting this pattern. You can see it here: https://serverlessland.com/patterns/apigw-sqs-cdk-typescript

Cheers!

**Pull request merged**